shared/cmd/flags.go

```diff
@@ -76,9 +76,15 @@ var (
 			"relay node and advertise their address via the relay node to other peers",
 		Value: "",
 	}
-	// P2PPort defines the port to be used by libp2p.
-	P2PPort = cli.IntFlag{
-		Name:  "p2p-port",
+	// P2PUDPPort defines the port to be used by discv5.
+	P2PUDPPort = cli.IntFlag{
+		Name:  "p2p-udp-port",
+		Usage: "The port used by discv5.",
+		Value: 12000,
```

Comment:
is this ok? both UDP and TCP default value is 12000

Reply:
There wouldn't be any issue specifying for both transports to use the same port although this could lead to port congestion and packet if a lot of data is being sent through either transports. It would be better to have them in separate ports, thinking about this again. Thanks, changing it now

Reply:
thanks